R/assertions.R

```diff
@@ -517,6 +517,8 @@ assert_filter_cond <- function(arg,
     return(invisible(arg))
   }
 
+  arg_name <- gsub("^.*\\((.*)\\)$", "\\1", arg_name)
```

Comment:
@bundfussr and @ddsjoberg  is there another way to do this using something from rlang? 
![image](https://github.com/pharmaverse/admiraldev/assets/10111024/8a21c73c-e2fa-4aac-b7f3-59f24aeab2e4)

Reply:
rlang has functions for working with calls, but not strings. If you have a call, you can use `rlang::call_args()` to get the arguments of the call.

``` r
my_call <- rlang::expr(rlang::enquo(data))
my_call
#> rlang::enquo(data)
```

``` r
rlang::call_args(my_call)
#> [[1]]
#> data
```

<sup>Created on 2024-05-17 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Reply:
I may be able to put it further in the function after call_args if we can't replace it in the arguments, but rlang::expr_text() returned 'my_expression' in place of 'filter_add' when generating the error. I thought that could work, but don't know what everyone else thinks.